Metadata/ListMetadataLoader.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AutomationBundle\Metadata;
+
+use Sulu\Bundle\AdminBundle\Metadata\ListMetadata\FieldMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\ListMetadata\ListMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\ListMetadata\ListMetadataLoaderInterface;
+use Sulu\Bundle\AdminBundle\Metadata\ListMetadata\XmlListMetadataLoader;
+use Sulu\Bundle\AdminBundle\Metadata\MetadataInterface;
+
+class ListMetadataLoader implements ListMetadataLoaderInterface
+{
+    /**
+     * @var XmlListMetadataLoader
+     */
+    private $xmlListMetadataLoader;
+
+    public function __construct(XmlListMetadataLoader $xmlListMetadataLoader)
+    {
+        $this->xmlListMetadataLoader = $xmlListMetadataLoader;
+    }
+
+    /**
+     * @param mixed[] $metadataOptions
+     */
+    public function getMetadata(string $key, string $locale, array $metadataOptions = []): ?MetadataInterface
+    {
+        if ('tasks' !== $key) {
+            return null;
+        }
+
+        if (!method_exists(FieldMetadata::class, 'setTransformerTypeParameters')) {
+            return null;
+        }
+
+        $list = $this->xmlListMetadataLoader->getMetadata($key, $locale, $metadataOptions);
+
+        if (!$list instanceof ListMetadata) {
+            return $list;
+        }
+
+        foreach ($list->getFields() as $field) {
+            if ('status' !== $field->getName()) {
+                continue;
+            }
+
+            $field->setType('icon');
+            $field->setVisibility('always');
```

Comment:
Should the status not also be displayed always also when not having the status with icons /cc @wachterjohannes

Reply:
you are right - @luca-rath can you add this?

Reply:
would be okay for me, but i wanted to keep it like it was before.

Reply:
I think its the most important information on the list so it should always be displayed